suite-common/wallet-config/src/networksConfig.ts

```diff
@@ -423,6 +423,7 @@ export const networks = {
         bip43Path: "m/44'/501'/i'/0'", // TODO(vl): accounts and paths
         decimals: 9,
         testnet: false,
+        chainId: 1399811149,
```

Comment:
so far only note for me - why do we need this?

Reply:
I checked through the codebase, I think that we don't.